etc/agent.properties

```diff
@@ -0,0 +1,44 @@
+###
+### Sample Agent configuration
+###
+
+#### Presto Manager configuration ###################################
+
+packaging             = rpm
+defaults.config-dir   = defaults/config
+defaults.catalog-dir  = defaults/catalog
+
+
+#### Airlift properties #############################################
+
+# node.id is autogenerated
+node.environment      = test
+http-server.http.port = 8081
+log.levels-file       = etc/log.properties
+discovery.uri         = http://localhost:8088
```

Comment:
In this file. The discovery port for both agent and controller should be the controller's HTTP port.